core/primitives-core/src/config.rs

```diff
@@ -262,9 +263,15 @@ pub struct ViewConfig {
     pub max_gas_burnt: Gas,
 }
 
+#[derive(Debug, Clone, Hash, PartialEq, Eq)]
+pub struct WeightedGas {
```

Comment:
This naming makes it easy to be confused with https://docs.rs/near-sdk/latest/near_sdk/struct.GasWeight.html

Also, we already have "used", "prepaid", "burnt", "burned", and "attached" gas as terms used throughout the code and it often makes technical communication difficult. E.g. the question "how much gas does this TX use" already has at least 3 correct answers, depending on how you define "used gas" and transaction. Weighted gas adds one more interpretations of "used gas". :see_no_evil: 

I don't know a better name, it's probably fine to keep it. But when writing the NEP (which we should start soon) we will need to make a final decision if we have an alternative naming or if we want to keep this.

Previously brainstormed ideas for what we can call weighted gas include:
- fumes or emissions (parameter weight defines how much emissions burning gas causes)
- cycles  / compute time / chunk space (naming it after the condition checked when we include receipts in a chunk)

Again, keeping the name is probably fine. But I wanted to write this down here to give you the full context and maybe you can see a way to minimize how complicated the story around gas is.

Reply:
As discussed on the NEP, renamed the whole struct into
```rust
struct ParameterCost {
  gas: Gas,
  compute: u64,
}
```

I also plan to introduce a dedicated type for compute cost.